jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/HugeResourceTest.java

```diff
@@ -114,11 +113,10 @@ public static void prepareStaticFiles() throws IOException
             String.format("FileStore %s of %s needs at least 30GB of free space for this test (only had %,.2fGB)",
                 baseFileStore, staticBase, (double)(baseFileStore.getUnallocatedSpace() / GB)));
 
-        makeStaticFile(staticBase.resolve("test-1M.dat"), MB);
-        makeStaticFile(staticBase.resolve("test-100M.dat"), 100 * MB);
-        makeStaticFile(staticBase.resolve("test-1G.dat"), GB);
-        // makeStaticFile(staticBase.resolve("test-4G.dat"), 4 * GB);
-        // makeStaticFile(staticBase.resolve("test-10G.dat"), 10 * GB);
+        makeStaticFile(staticBase.resolve("test-1m.dat"), MB);
+        makeStaticFile(staticBase.resolve("test-1g.dat"), GB);
+        // makeStaticFile(staticBase.resolve("test-4g.dat"), 4 * GB);
+        // makeStaticFile(staticBase.resolve("test-10g.dat"), 10 * GB);
```

Comment:
The reason for testing 4GB and 10GB were because of various filesystem bugs (the 2GB threshold and the 8GB threshold in various FileSystem APIs).

Reply:
I've just restored the `HugeResourceTest.java` file to exactly what it was before Greg disabled it. But I think your comment is worth adding to the code.

Reply:
Okay, rethrowing as `AssertionError` then.

Reply:
But 4g and 10g are still commented out?